integration-cli/utils_unix_test.go

```diff
@@ -0,0 +1,19 @@
+// +build linux freebsd solaris openbsd
+
+package main
+
+import (
+	"github.com/docker/docker/pkg/parsers/kernel"
+)
+
+// GetKernelVersion gets the current kernel version.
+func GetKernelVersion() *kernel.VersionInfo {
+	v, _ := kernel.ParseRelease(testEnv.DaemonInfo.KernelVersion)
+	return v
+}
+
+// CheckKernelVersion checks if current kernel is newer than (or equal to)
+// the given version.
+func CheckKernelVersion(k, major, minor int) bool {
```

Comment:
Since this is only used in a single place please move these 2 functions to immediately after the test which uses them.